src/components/ContactUsForm/Contact_Us.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./Contact_Us.css";
+
+const ContactUs = () => {
+  return (
+    <div className="contact-us-container">
+      <div className="contact-form">
+        <h1>Contact Us</h1>
+        <form>
+          <label>Full Name</label>

```

Comment:
now, we can keep this as what it was before i.e. 

```
<input placeholder="Full Name" />
 <input placeholder="Email"/>   
```

(no need for label fields with input with current styles :)